template/template.go

```diff
@@ -166,7 +166,7 @@ func NewTemplateExpander(
 				return html_template.HTML(text)
 			},
 			"match":     regexp.MatchString,
-			"title":     strings.Title, //nolint:staticcheck
+			"title":     strings.Title,
```

Comment:
This one is weird because `strings.Title` is deprecated; it should get a warning.

Reply:
@bboreham Guessing this was a bug in v1.60.1, since this is caught again in v1.60.2.